### PR TITLE
PORTS GRILLS AND BBQ RIBS

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by MaterialisticThings** [comment]
+1

**Comment by karmaisblackandbluepilled** [comment]
+1
COOM AT HIGH SPEEDS

**Comment by Fikou** [comment]
borgar

**Comment by cris5311** [comment]
+1

**Comment by Manly-Dwarf** [comment]
+1

File: code/game/objects/items/stacks/sheets/mineral.dm [hunk 1]
```diff
@@ -15,8 +15,8 @@ Mineral Sheets
 	Others:
 		- Adamantine
 		- Mythril
-		- Enriched Uranium
```
Comment (ghost) [inline code/game/objects/items/stacks/sheets/mineral.dm:18]: Beeg Beeg thonk. Make sure this isn't an error.

File: code/modules/food_and_drinks/drinks/drinks.dm [hunk 2]
```diff
@@ -548,6 +548,13 @@
 	list_reagents = list(/datum/reagent/consumable/grey_bull = 20)
 	foodtype = SUGAR | JUNKFOOD
 
+/obj/item/reagent_containers/food/drinks/soda_cans/monkey_energy
```
Comment (ghost) [inline code/modules/food_and_drinks/drinks/drinks.dm:551]: Is this part of the PR?

**Comment by ghost** [comment]
+1

File: code/modules/food_and_drinks/drinks/drinks.dm (hunk 2 above)
Comment (ghost) [reply code/modules/food_and_drinks/drinks/drinks.dm:551]: yeah

File: code/game/objects/items/stacks/sheets/mineral.dm (hunk 1 above)
Comment (ghost) [reply code/game/objects/items/stacks/sheets/mineral.dm:18]: just a comment, who cares?

Comment (ghost) [reply code/game/objects/items/stacks/sheets/mineral.dm:18]: Is this part of the original PR? If not, it's better to fix it.

Comment (ghost) [reply code/game/objects/items/stacks/sheets/mineral.dm:18]: firefox_3JjU5CpoBT

**Review by ghost — APPROVED** [review]
Aside from the funky dme changes, this is good to go

**Comment by ghost** [comment]
Also, you did one up me with this one

**Pull request merged**